recipes/samtools/develop/meta.yaml

```diff
@@ -11,7 +11,7 @@ about:
   summary: C library for high-throughput sequencing data formats.
 
 build:
-  number: 2
+  number: 3
```

Comment:
It's intentional to reset htslib build to `0` and increment samtools build to `3`?

Reply:
Yes - as samtools version is staying the same but the new build is relying on a more recent htslib version.